macros/REST_GenerateDataSet.C

```diff
@@ -0,0 +1,29 @@
+#include "TRestDataSet.h"
+#include "TRestTask.h"
+
+#ifndef RestTask_GenerateDataSet
+#define RestTask_GenerateDataSet
+
+//*******************************************************************************************************
+//*** Description: This macro will launch the generation of datasets defined
+//*** inside a particular RML file `datasets.rml` that contains the dataset
+//*** definitions. The second argument will allow to specify the datasets
+//*** to be generated from the existing ones inside `dataset.rml`.
+//***
+//*** --------------
+//*** Usage: restManager GenerateDataSet datasets.rml set1,set2,set3
+//***
+//*******************************************************************************************************
+
+Int_t REST_GenerateDataSet(std::string inputRML, std::string datasets) {
```

Comment:
From my experience, I think it would be more useful to generate a dataset using the file pattern. E.g.
```
Int_t REST_GenerateDataSet(const std::string &inputRML, const std::string &datasetName, const std::string &filePattern, const std::string &outputFileName) {
    TRestDataSet d(inputRML.c_str(), datasetName.c_str());
    d.SetFilePattern(filePattern);
    d.GenerateDataSet();
    d.Export(outputFileName);
    return 0;
}
```

Reply:
The idea is to generate several datasets at once, then the output file name takes the name of the dataset with the `dataset_` root.

Reply:
On top of that is that each dataset has a different `filePattern`.

I am using it as follows with the following file: https://github.com/iaxo/iaxo-simulations/blob/master/montecarlo/signal/sensitivity/dataset.rml

```
restManager GenerateDataSet BabyIAXO_XMM_mm_Vacuum_0meV,BabyIAXO_XMM_mm_Vacuum_1meV,BabyIAXO_XMM_mm_Vacuum_10meV,BabyIAXO_XMM_mm_Vacuum_100meV,BabyIAXO_XMM_mm_Vacuum_1eV
```

To generate:

<img width="1049" alt="Screenshot 2023-05-25 at 12 02 53" src="https://github.com/rest-for-physics/framework/assets/12447509/1140c93d-9f71-414a-88b8-36311063c9d5">

Reply:
Then I would suggest to start the dataset with `D` contrary to the TRestRuns which typically starts with `R`

Reply:
Ok, I could start with upper case letter, this was my default choice in the past. I think @lobis prefers start with lowercase.

Reply:
Right now I don't recall stating any preference, but yes in general I use lowercase names for all file names (easier to type in my opinion) but it's up to personal preference 😃